apis/r/NEWS.md

```diff
@@ -1,3 +1,7 @@
+# 1.9.1
+
+* This release contains a single Python-only mod
```

Comment:
It's fine but a few added words such 'Python-only modification for its build process' are more informative